src/components/DefineCompany/AddNameTranslation.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div id="name-translation">
+    <!-- Name Translation form -->
+    <v-form v-model="nameTranslationForm" ref="ntForm" name="name-translation-form">
+      <v-row>
+        <v-col class="pb-0">
+          <v-text-field
+            filled
+            persistent-hint
+            label="Enter Name Translation"
+            v-model="nameTranslation"
+            :rules="nameTranslationRules">
```

Comment:
Would be good practise to give this text field an id (or class), for testing at least.

Reply:
You read my mind, implemented while i was writing the tests :) Will be in next commit!